brambling/models.py

```diff
@@ -299,6 +302,30 @@ def create_defaults(app_config, **kwargs):
             ])
 
 
+class OrganizationMember(models.Model):
+    EDIT = '1-edit'
+    VIEW = '2-view'
+    OWNER = '0-owner'
```

Comment:
They allow ordering by the role field to return a list in order of descending permissions.